olive/evaluator/accuracy.py

```diff
@@ -62,6 +61,13 @@ def _metric_config_from_torch_metrics(cls):
     def _default_config(cls) -> Dict[str, ConfigParam]:
         return cls._metric_config_from_torch_metrics()
 
+    @staticmethod
+    def prepare_tensors(preds, target, dtypes=torch.int):
+        dtypes = dtypes if isinstance(dtypes, (list, tuple)) else [dtypes, dtypes]
```

Comment:
could we just use the dtype instead of [dtypes, dtypes]?

Reply:
The idea was to support either a len2 tuple/list or a single type. So that the types for preds and tensor can be different if wanted. 
I am using the `[dtypes, dtypes]` part to make the logic simpler (no additional conditions) for the following lines that refer to `dtypes[0]` and `dtypes[1]`.